BootloaderCorePkg/Tools/BuildUtility.py

```diff
@@ -862,7 +862,11 @@ def check_for_git():
 
 def check_for_slimbootkeydir():
     if not os.path.exists(os.environ.get('SBL_KEY_DIR')):
-        raise Exception ("SBL_KEY_DIR is not valid. Set correct SBL_KEY_DIR path !!")
+        print ("!!! PRE_REQUISTE: PATH TO SBL_KEY_DIR HAS TO BE SET WITH SBL KEY DIRECTORY !!! \n"

```

Comment:
Let us separate 2 cases
- SBL_KEY_DIR is not set.  So prompt user the default PATH is used.
- SBL_KEY_DIR is a valid path, but there is no keys in it.   Prompt the msg below.

Reply:
Addressed both cases.